pages/platform/data-processing/40_TUTORIAL_calculate_pi/guide.en-asia.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Python - Calculating π number with Apache Spark
+slug: pi-spark
+excerpt: Calculting π number is a basic example about how to use Apache Spark. Let’s find out how to do it !
+section: Tutorials
+order: 1
+---
+
+**Last updated 30<sup>th</sup> April, 2020**
+
+## Objective
+This guide  gives you a basic example about using Apache Spark and OVHcloud Data Processing.
+We will first explain how to calculte π number, then use a naive method and OVHcloud Data Processing.
+
+We want to calculate the value of π (pi) with the Monte-Carlo method. It is based on the following principle: we use a square which side equals 1 and a circle of radius 1 and center (0,0) as shown in the graph below. Then take a random point inside this square, the probability that it is in the circle equals to π/4.
+
+![Monte Carlo Graph](images/monte_carlo_graph.png){.thumbnail}
+
+This probabilty can be estimated by picking a high number of random points and then dividing the number of points within the circle by the total number of points.
+
+
+## Requirements 
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager){.external}
+- An OVHcloud account 
+- A cloud project in your OVHcloud account (see [How to create a cloud project](../../public-cloud/getting_started_with_public_cloud_logging_in_and_creating_a_project){.external} for details.)
+- Data Processing activated (see [How to activate the Data Processing](../activation){.external} for details.)
```

Comment:
How to activate the Data Processing service
-> Change this in all pages as it seems to be always wrong even in pages that are not in this PR